src/backend/web/graphql/index.js

```diff
@@ -22,6 +22,7 @@ module.exports.typeDefs = graphql`
     author: String
     url: String
     posts: [Post]
+    user: String
```

Comment:
I don't think we need this here.  We shouldn't need to expose this to the front end.  If we end up needing it for something in the future, let's add it then.